saleor/graphql/schema.graphql

```diff
@@ -9796,6 +9796,7 @@ type Invoice implements ObjectWithMetadata & Job & Node {
 
   """URL to download an invoice."""
   url: String
+  orderId: ID
```

Comment:
Why do we need this field? I think we can access the order already using other new fields that this PR adds. Also in the schema `Invoice` seems to be always embedded within an order object, so we can reference the parent to get the order.